epub33/core/index.html

```diff
@@ -9423,10 +9423,20 @@ <h3>Recommendations</h3>
 					creators should also allow users to opt out of tracking, when feasible, and provide users the
 					ability to manage and delete any data that is collected about them.</p>
 
-				<p>Content authors also need to consider the inadvertent collection of information about users. Linking
-					to content on a publisher's web site, or remotely hosting resources on their servers, can lead to
+				<p>EPUB creators also need to consider the inadvertent collection of information about users. Linking to
+					content on a publisher's web site, or remotely hosting resources on their servers, can lead to
 					profiling users, especially if unique tracking identifiers are added to the URLs.</p>
 
+				<p>When collecting and storing user information within an EPUB publication (e.g., through the use of <a
+						data-cite="html#dom-document-cookie">cookies</a> and <a data-cite="html#webstorage">web
+						storage</a> [[?html]]), EPUB creators need to consider to potential for data theft by other EPUB
+					publications on a reading system. Although [[epub-rs-33]] introduces a <a
+						data-cite="epub-rs-33#sec-container-iri">unique origin requirement</a> for EPUB publications,
+					which limits the potential for attacks, older reading systems may allow all EPUB publications access
```

Comment:
"older or non-conformant reading systems"?

Reply:
I've moved the examples to a parenthetical at the end to declutter a bit. It now says:

> Although [[epub-rs-33](#bib-epub-rs-33)] introduces a [unique origin requirement](https://www.w3.org/TR/epub-rs-33/#sec-container-iri) for EPUB publications, which limits the potential for attacks, there is still a risk that reading systems will allow EPUB publications access to shared persistent storage (e.g., older reading systems that have not been updated and non-conforming newer reading systems).